APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -750,6 +753,28 @@ - (void) logOutNotification: (NSNotification*) __unused notification
     [self showOnBoarding];
 }
 
+- (void) logOutAndGoToSignIn
+{
+    self.dataSubstrate.currentUser.signedUp = NO;
+    self.dataSubstrate.currentUser.signedIn = NO;
+    [APCKeychainStore removeValueForKey:kPasswordKey];
+    [self.tasksReminder updateTasksReminder];
+    [self showOnBoarding];
+
+    
+    // Check if the showOnBoarding is showing the study Overview, because then we can enabled sign in to show right afterwards
+    // This will work for vanilla implementations of AppCore, but may need additional implementation in some apps
+    UIViewController* topVc = ((UINavigationController*)self.window.rootViewController).topViewController;
+    if([topVc isKindOfClass:[APCStudyOverviewCollectionViewController class]])
+    {
+        ((APCStudyOverviewCollectionViewController*)topVc).transitionToSignInOnLoad = YES;
+    }
+    else
+    {
+        NSLog(@"Error: We were unable to automatically transition to the sign-in screen. This is probably because the study overview screen has been replaced by your app. One way to add this feature into your app is to register for the APCUserLogOutAndGoToSignInNotification in your AppDelegate, and modify the behavior of the method showOnBoarding to transition straight to the sign-in screen");
+    }
```

Comment:
In mPower, showOnboarding does have custom animations, and this still works fine, so that shouldn't be a problem.

I'm not really sure I understand your showLogin suggestion without more details.